src/openms/include/OpenMS/KERNEL/SpectrumHelper.h

```diff
@@ -34,6 +34,10 @@
 
 #pragma once
```

Comment:
Is there a python interface to these methods?  If not, one will be needed.

Reply:
No, there isn't.
Since there is no class involved, I should look for how to implement it (there is no .pxd file for this file, yet)